### ISSUE TITLE
Hi, Thank you  for your work.You divide the training set of youtubevis into training set and validation set according to a certain proportion. How is it divided? Is there any code for reference?

### ISSUE THREAD

**Issue by Huang-erying** [haotian-liu/yolact_edge#183]
**Describe the bug**

**To Reproduce**
Steps to reproduce the behavior (full command you are using).

**Expected behavior**

**Full logs**
If applicable, add full logs to help explain your problem.

**Environment:**
 - OS: [e.g. Ubuntu 18.04]
 - GPU: [e.g. RTX 2080 Ti *4]
 - CUDA Version [e.g. 10.2]

**Additional context**

**Comment by haotian-liu** [comment]
We divide the training split into two train-val splits with a 85%-15% ratio (1904 and 334 videos).
